From: Maykel Moya <mmoya@mmoya.org>
Date: Mon, 22 Oct 2012 11:56:29 +0200
Subject: Fix Makefile

* Do not install thcping6 twice.
* Rename thc-ipv6.8 manpage to attack-toolkit6.8.
* Remove language extension from scripts in $PATH.
* Set prefix to /usr.
* Don't link against OpenSSL libraries as there is not an
  exemption yet.
---
 Makefile |   11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

--- thc-ipv6.orig/Makefile
+++ thc-ipv6/Makefile
@@ -11,7 +11,7 @@
 LIBS=thc-ipv6-lib.o
 STRIP=echo
 
-PREFIX=/usr/local
+PREFIX=/usr
 MANPREFIX=${PREFIX}/share/man
 
 all:	$(LIBS) $(PROGRAMS) dnssecwalk dnsdict6 thcping6
@@ -20,7 +20,7 @@
 	$(CC) $(CFLAGS) -o $@ $^
 
 dnsdict6:	dnsdict6.c
-	$(CC) $(CFLAGS) -o $@ $^ -lpthread -lresolv
+	$(CC) $(CFLAGS) -o $@ $^ $(LDFLAGS) $(CPPFLAGS) -lpthread -lresolv
 
 thcping6:	thcping6.c $(LIBS)
 	$(CC) $(CFLAGS) -o $@ $^ $(LDFLAGS) -lrt
@@ -33,9 +33,12 @@
 
 install: all strip
 	install -m0755 -d ${DESTDIR}${PREFIX}/bin
-	install -m0755 $(PROGRAMS) dnsdict6 thcping6 dnssecwalk *.sh ${DESTDIR}${PREFIX}/bin
+	install -m0755 $(PROGRAMS) dnsdict6 ${DESTDIR}${PREFIX}/bin
+	install -m0755 dos_mld.sh ${DESTDIR}${PREFIX}/bin/dos_mld
+	install -m0755 extract_hosts6.sh ${DESTDIR}${PREFIX}/bin/extract_hosts6
+	install -m0755 extract_networks6.sh ${DESTDIR}${PREFIX}/bin/extract_networks6
 	install -m0755 -d ${DESTDIR}${MANPREFIX}/man8
-	install -m0644 -D thc-ipv6.8 ${DESTDIR}${MANPREFIX}/man8
+	install -m0644 -D thc-ipv6.8 ${DESTDIR}${MANPREFIX}/man8/attack-toolkit6.8
 
 clean:
 	rm -f $(PROGRAMS) dnsdict6 thcping6 dnssecwalk $(LIBS) core DEADJOE *~
